File: python/sdk/moderation_video_demo.py
# -*- coding:utf-8 -*-
from ais_sdk.gettoken import get_token
from ais_sdk.utils import encode_to_base64
from ais_sdk.moderation_video import moderation_video

if __name__ == '__main__':
    user_name = '******'
    password = '******'
    account_name = '******'  # the same as user_name in commonly use

    user_name = 'wwfnwg'
    password = 'Huawei@123'
    account_name = 'wwfnwg'  # the same as user_name in commonly use

    demo_data_url = 'https://obs-test-llg.obs.cn-north-1.myhwclouds.com/bgm_recognition'
    token = get_token(user_name, password, account_name)

    # call interface use the url
    result = moderation_video(token, demo_data_url, 5)
    print result

File: php/moderation_video_demo.php
<?php
/**
 * 语音识别服务的使用示例
 */
require "./ais_sdk/gettoken.php";
require "./ais_sdk/moderation_video.php";
require "./ais_sdk/utils.php";

/**
 * token 方式请求
 */
$username = "********";      // 配置用户名
$password = "********";      // 密码
$domainName = "*********";   // 配置用户名
$regionName = "********";    // 配置区域信息

$demo_data_url = "https://obs-test-llg.obs.cn-north-1.myhwclouds.com/bgm_recognition";

$token = gettoken($username, $password, $domainName, $regionName);

$result = moderation_video($token, $demo_data_url, 5, array("terrorism", "porn", "politics"));
echo json_encode($result);
echo "\n";
/**
 * ak,sk 方式请求
 */

$app_key = "*************";
$app_secret = "*************";

// obs的url方式请求
$result = moderation_video_aksk($app_key, $app_secret, $demo_data_url, 5, array("terrorism", "porn", "politics"));
echo json_encode($result);


